Ryujinx/_schema.json

```
@@ -636,9 +636,24 @@
               "default": "O"
             }
           }
+        },
+        "hotkeys": {
+          "$id": "#/properties/keyboard_controls/properties/hotkeys",
+          "type": "object",
+          "title": "Hotkey Controls",
+          "required": [
+            "toggle_vsync"
+          ],
+          "properties": {
+            "button_zr": {
```

This is incorrect, should be `toggle_vsync`
```suggestion
            "toggle_vsync": {
```